pkg/kube/inject/inject.go

```diff
@@ -473,6 +474,24 @@ func InjectionData(sidecarTemplate, valuesConfig, version string, typeMetadata *
 			return nil, "", merr
 		}
 	}
+
+	cluster := values.Map("global").Map("multiCluster").String("clusterName")
+	network := values.Map("global").String("network")
```

Comment:
is there a structured object? this is error-prone if we change something later

Reply:
We could unmarshal to the proto struct but:
1. that doesn't play nicely with the templates
2. the templates are just as fragile as this code - these two things are pretty tightly coupled.